src/features/events_list/components/EpisodeTimelineToggle/EpisodeTimelineToggle.tsx

```diff
@@ -1,17 +1,31 @@
+import { useRef, useState } from 'react';
 import { useAtom } from '@reatom/react';
+import { Trans } from 'react-i18next';
 import { Play24, PlayActive24 } from '@konturio/default-icons';
+import { Tooltip } from '@konturio/ui-kit';
 import { episodesPanelState } from '~core/shared_state';
 import s from './EpisodeTimelineToggle.module.css';
 
 export function EpisodeTimelineToggle({ isActive }) {
   const [panelState, panelStateActions] = useAtom(episodesPanelState);
 
+  const ref = useRef(null);
+  const [showTooltip, setShowTooltip] = useState(false);
+
   return (
-    <span
-      className={s.timelineIcon}
-      onClick={panelState.isOpen ? panelStateActions.close : panelStateActions.open}
-    >
-      {isActive && panelState.isOpen ? <PlayActive24 /> : <Play24 />}
-    </span>
+    <>
+      <span
+        ref={ref}
+        className={s.timelineIcon}
+        onClick={panelState.isOpen ? panelStateActions.close : panelStateActions.open}
+        onMouseEnter={() => setShowTooltip(true)}
+        onMouseLeave={() => setShowTooltip(false)}
+      >
+        {isActive && panelState.isOpen ? <PlayActive24 /> : <Play24 />}
+      </span>
+      <Tooltip open={showTooltip} hoverBehavior>
+        <Trans i18nKey="event_list.open_timeline_button">Open timeline</Trans>
```

Comment:
hm, <Trans /> should only be used if there is html content inside translation string
here it seems that you need just {i18n.t("event_list.open_timeline_button")}